src/unittests/f32_interval.spec.ts

```diff
@@ -1087,7 +1088,9 @@ g.test('lengthIntervalScalar')
     [
       // Some of these are hard coded, since the error intervals are difficult to express in a closed human readable
       // form due to the inherited nature of the errors.
-      {input: 0, expected: kAny },  // ~0
+      //
+      // length(0) = kAny, because length uses sqrt, which is defined as 1/inversesqrt
```

Comment:
I find the fact that `length(0)` isn't always `0` really surprising. Is this right @dneto0 ?

Reply:
I agree it looks weird, I actually spent a non-trivial amount of time investigating this when I start writing this PR to make sure it was correct, I thought it was a bug in my code originally.

My understanding is that this is a consequence of the inherited accuracy.

length's accuracy is defined as `sqrt(x*x)`
square root's accuracy is defined as `1.0 / inverseSqrt(x)`
inverse square root's accuracy is defined as 2 ULP from truth, which is stated as `Returns the reciprocal of sqrt(e)`

So at x = 0 you have at the bottom of the inherited accuracy 2 ULP from 1/sqrt(0*0) which becomes 1/0, which is an undefined value, which is where `kAny` comes in and it propagates from there.

@dneto0 can you confirm that this is the intended interpretation? or correct me, since this affects a number of built-ins.

Reply:
Yes, that's the right interpretation.